src/list/GameList.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+import Link from 'next/link';
+import Button from '../ui/Button';
+
+const ListTitle = styled('span')`
+    font-size: 20px;
+    font-weight: bold;
+    text-align: center;
+    margin: 0 4px;
+    text-align: center;
+    margin: 10px 4px;
+    background-color: purple;
+    border-radius: 3px;
+    color: white;
+    line-height: 50px;
+`;
+
+const ListDataContainer = styled('div')`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    max-height: 500px;
+    overflow-y: scroll;
+    overflow-x: hidden;
+`;
+
+const ListContainer = styled('div')`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    border: 1px solid purple;
+    border-radius: 3px;
+`;
+
+class GameList extends Component {
+    state = {
+        list: [],
+        loaded: false,
+    };
+
+    static propTypes = {
```

Comment:
I'd rather have the propTypes added at the end of the file.
`GameList.propTypes = ..`
This way the propTypes is in the same place for class component and function component.